### ISSUE TITLE
Covert GBT Regression Score To Probability

### ISSUE THREAD

**Issue by zhangpengshan** [ShifuML/shifu#254]
Now GBT computing score is not in [0, 1], -0.124, 1.546 are found in output scores. Need a coverter to covert to probability.

Check scikit-learn:
https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/gradient_boosting.py#L519

Or H2O for reference.

**Comment by zhangpengshan** [comment]
Set it to high priority, find a better sigmoid function

**Comment by zhangpengshan** [comment]
Using basic sigmoid funtion, user needs to set:  "gbtConvertToProb" : to true,

  "evals" : [ {
	    "name" : "Eval1",
	    "dataSet" : {
	      "source" : "HDFS",
	      "dataPath" : "EvalSet1",
	      "dataDelimiter" : "|",
	      "headerPath" : "EvalSet1/.pig_header",
	      "headerDelimiter" : "|",
	      "filterExpressions" : "",
	      "weightColumnName" : "",
	      "targetColumnName" : "diagnosis",
	      "posTags" : [ "M" ],
	      "negTags" : [ "B" ],
	      "missingOrInvalidValues" : [ "", "*", "#", "?", "null", "~" ],
	      "metaColumnNameFile" : "columns/meta.column.names",
	      "categoricalColumnNameFile" : "columns/categorical.column.names"
	    },
	    "performanceBucketNum" : 10,
            "gbtConvertToProb" : true,
	    "performanceScoreSelector" : "mean",
	    "scoreMetaColumnNameFile" : "columns/Eval1score.meta.column.names",
	    "customPaths" : { }
	  } ]

**Comment by zhangpengshan** [comment]
Reopen this issue as a good regression formula should be find in production
